cocos/editor-support/spine/SkeletonRenderer.cpp

```diff
@@ -98,7 +98,7 @@ SkeletonRenderer::~SkeletonRenderer () {
 	spSkeleton_dispose(_skeleton);
 	if (_atlas) spAtlas_dispose(_atlas);
 	if (_attachmentLoader) spAttachmentLoader_dispose(_attachmentLoader);
-	delete _worldVertices;
+	delete [] _worldVertices;
```

Comment:
According Spine runtime license, cocos2d-x can not modify the codes. Could you please send a PR to [Spine runtime repo](https://github.com/EsotericSoftware/spine-runtimes) ? I will merge it if it is accepted by Spine runtime.

Thanks.